build-system/tasks/check-types.js

```diff
@@ -159,6 +160,33 @@ const TYPE_CHECK_TARGETS = {
     externGlobs: [CORE_EXTERNS_GLOB, 'build-system/externs/*.extern.js'],
   },
 
+  /*
+   * Ensures that all files in src and extensions pass the specified set of errors.
+   */
+  'low-bar': {
+    entryPoints: ['src/amp.js'],
+    extraGlobs: ['{src,extensions}/**/*.js'],
+    onError(msg) {
+      const lowBarErrors = [
```

Comment:
Suggestion: consider putting this at the top with PRIDE_GLOBS etc. I don't feel strongly about it though

Reply:
I prefer co-location considering the variable is used in a single location.